### PR TITLE
Contributing missingno

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: recipes/missingno/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - import missingno
```
Comment (synapticarbors) [inline recipes/missingno/meta.yaml]: Drop the `import` and just use:

```
imports:
   - missingno
```

File: recipes/missingno/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/missingno/meta.yaml]: It looks like this package requires `numpy`, `scipy`, `pandas`, `matplotlib` and `seaborn` at runtime.

File: recipes/missingno/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - import missingno
+
+about:
+  home: https://github.com/ResidentMario/missingno
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
```
Comment (synapticarbors) [inline recipes/missingno/meta.yaml]: It looks like `LICENSE.md` is not packaged due to a missing entry in a `MANIFEST.in` file. Please see:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

and please put in an issue or a pull request upstream for future releases.

Comment (ericmjl) [inline recipes/missingno/meta.yaml]: @synapticarbors thanks for the note. Just made the change, please look out for next push.

File: recipes/missingno/meta.yaml (hunk 1 above)
Comment (ericmjl) [inline recipes/missingno/meta.yaml]: My bad; it's been a while since I last edited one of these.

File: recipes/missingno/meta.yaml (hunk 2 above)
Comment (ericmjl) [inline recipes/missingno/meta.yaml]: My bad, boils down to bad memory about what needs to be done. Thanks for double-checking!

File: recipes/missingno/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - missingno
+
+about:
+  home: https://github.com/ResidentMario/missingno
+  license: MIT
+  license_family: MIT
+  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE
```
Comment (synapticarbors) [inline recipes/missingno/meta.yaml]: Maybe the linter is failing due to this needing to be wrapped in single quotes?

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Comment (ericmjl) [inline recipes/missingno/meta.yaml]: hmmm.... I tried changing that, it doesn't work either.

Comment (synapticarbors) [inline recipes/missingno/meta.yaml]: If you look at the link, it's not just switching single for double quotes around `RECIPE_DIR`. It's that the whole thing is inside the quotes. Again, I'm not sure if this is the reason that the linter is complaining.

Comment (ericmjl) [inline recipes/missingno/meta.yaml]: My apologies; I was distracted when I made that change. Let me try with the correct way.

**Comment by ericmjl** [comment]
@synapticarbors I think you were right re: formatting.

**Pull request merged**